packages/rich-text/src/RichTextEditor.styles.ts

```diff
@@ -31,6 +31,7 @@ export const styles = {
     // We need to reset LIC style due to conflicts between PARAGRAPH styles
     'ul > li > div': {
       margin: 0,
+      marginBottom: '1.25rem !important',
```

Comment:
* the margin here is a bit too much. half of that is probably enough.
* How about applying the margin to the void entity blocks directly instead?

Reply:
Found a way to do it "better".
That's the same margin of a block at root level. Do we want less space in a list?